### DISCUSSION TITLE
Bitrate over 128 by mp3

### DISCUSSION THREAD

**Discussion by PrinceIcedale** [audiamus/AaxAudioConverter#145] [Ideas]
Is would also be possible to select bitrates of 192, 224 & 320. If it is not too much effort.

Thanks

**Comment by audiamus** [comment]
That wouldn't make much sense. The highest bitrate observed in the original AAC audio is 128 kb/s.

**Comment by PrinceIcedale** [comment]
I previously encoded with aax3mp3 and found that the quality was better with 320 than with 128, so I always encoded with 320. The sound was a bit clearer and easier to understand.

**Comment by PrinceIcedale** [comment]
Since this tool is class programmed, it would be great if it works. It would only be an option extension.

**Comment by audiamus** [comment]
I'm sorry, I won't change that. The limitation is by intention. Available options shouldn't mislead users.

**Comment by PrinceIcedale** [comment]
Is ok, but thanks for the quick discussion and especially I will use the program from now on.

**Comment by LeeBinder** [comment]
choose m4a 128 kb/s which is native Audible aax (which also uses 44.1 kHz sample rate inside their m4b)
